robottelo.properties.sample

```diff
@@ -127,3 +127,13 @@ test.virtual_machines_list=127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1
 # Class `ConcurrentTestCase` and its subclasses use this setting when
 # computing statistics of each performance test case, grouped in buckets.
 csv.num_buckets=10
+
+# Target repository names to be synchronized by Pulp.
+test.target_repos=Repo Example 1,Repo-Example-2
+
+# Number of times to repeat synchronization on each repository
+test.num_syncs=3
+
+# Parameter for deciding whether conduct initial sync or resync
+# 0 denotes resync; 1 denotes initial sync
+test.is_initial_sync=0
```

Comment:
You're not limited to integers. Why not provide a string?

    test.test_type='resync'
    test.test_type='sync'